spec/overcommit/hook/prepare_commit_msg/replace_branch_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+describe Overcommit::Hook::PrepareCommitMsg::ReplaceBranch do
```

Comment:
This file lacks `repo` tests. Let me know if these are required for this PR to be merged.

Reply:
This is fine. As long as we have rigorous tests for the `PrepareCommitMsg::Base` class which validate the behavior of our commit message modifier helpers and the `commit_message_source` / `commit_message_source_ref` helpers, we can stub those in hook implementation tests.